app/controllers/profiles_controller.rb

```diff
@@ -1,16 +1,13 @@
-
 class ProfilesController < ApplicationController
-  include LolConnections
   before_action :require_user, only: [:show]
   before_action :check_or_set_user, except: [:index]
-  before_action :verify_summoner_name_and_stats, only: [:create]
 
   def index
   end
 
   def show
     @user ||= current_user
-    featured_stats_service = FeaturedStatsService.new(@user)
+    featured_stats_service = FeaturedStatsService.new(@user.name)
```

Comment:
@danfstucky I was trying to login but I kept getting Lol::NotFound error on this line. Are you able to login to any profile?